README.md

```diff
@@ -13,62 +13,126 @@
+
+By default, most variables' values are assumed to be `true` (with exception of `using_credit_card_payment_txn`). In other to enable or disable the relevant functionality in the package, you will need to add the relevant variables:
+
+```yml
+vars:
+  quickbooks_source:
+    using_address: false # disable if you don't have addresses in QuickBooks
+    using_bill: false # disable if you don't have bills or bill payments in QuickBooks
+    using_credit_memo: false # disable if you don't have credit memos in QuickBooks
+    using_department: false # disable if you don't have departments in QuickBooks
+    using_deposit: false # disable if you don't have deposits in QuickBooks
+    using_estimate: false # disable if you don't have estimates in QuickBooks
+    using_invoice: false # disable if you don't have estimates in QuickBooks
+    using_invoice_bundle: false # disable if you don't have estimates in QuickBooks
+    using_journal_entry: false # disable if you don't have estimates in QuickBooks
+    using_payment: false # disable if you don't have estimates in QuickBooks
+    using_refund_receipt: false # disable if you don't have estimates in QuickBooks
+    using_transfer: false # disable if you don't have estimates in QuickBooks
+    using_vendor_credit: false # disable if you don't have estimates in QuickBooks
+    using_sales_receipt: false # disable if you don't have estimates in QuickBooks
+    using_credit_card_payment_txn: true # enable if you want to include credit card payment transactions in your staging models
+```
+
+## (Optional) Step 5: Additional Configurations
+<details><summary>Expand for configurations</summary>
+
+### Changing the Build Schema
+By default this package will build the QuickBooks staging models within a schema titled (<target_schema> + `_quickbooks_staging`) and QuickBooks final models within a schema titled (<target_schema> + `_quickbooks`) in your target database. If this is not where you would like your modeled QuickBooks data to be written to, add the following configuration to your `dbt_project.yml` file:
 
 ```yml
 # dbt_project.yml
 
 ...
-config-version: 2
+models:
+    quickbooks:
+      +schema: my_new_schema_name # leave blank for just the target_schema
+    quickbooks_source:
+      +schema: my_new_schema_name # leave blank for just the target_schema
+```
 
+### Change the source table references
+If an individual source table has a different name than the package expects, add the table name as it appears in your destination to the respective variable:
+> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_quickbooks/blob/main/dbt_project.yml) variable declarations to see the expected names.
+    
+```yml
 vars:
-    quickbooks_database: your_database_name
-    quickbooks_schema: your_schema_name
-```
+    quickbooks_<default_source_table_name>_identifier: your_table_name 
+``` 
```

Comment:
Same comment from my re-review of the source package. Can you move the change build schema and change source table references to be the last ones in this section. 

I feel the union feature, cash flow ordering, and balance sheet/income statement ordering are more important than the schema and source changes. I feel it would be more impactful for the more relevant items to be higher in this section.

Reply:
Moved it!